Project.toml

```
@@ -33,6 +34,7 @@ SymEngine = "0.8"
 SymPy = "1"
 Symbolics = "4, 5"
 julia = "1.6"
+Combinatorics = "1.0.2"
```

```suggestion
Combinatorics = "1"
```
Do you really require the patch release 1.0.2? If not, the more general version 1 should be fine